server/lib/flow-event.js

```diff
@@ -23,7 +24,98 @@ var HOSTNAME = os.hostname();
 var MAX_DATA_LENGTH = 100;
 var VERSION = 1;
 
-module.exports = function (event, data, request) {
+const FLOW_BEGIN_EVENT_TYPES = /^flow\.[a-z_-]+\.begin$/;
```

Comment:
Fwiw, as a general rule, I've come to prefer not to do that out of consideration for the reviewer. Sometimes I find it quite taxing to review diffs that contain lots of incidental changes to lines that wouldn't otherwise be affected by the proposed change. It can obscure the main proposal, when it gets of hand.

But anyway, yes, I'll update these.